specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlServer.json

```diff
@@ -0,0 +1,1975 @@
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "Remove workspace managed sql Server's vulnerability assessment": {
+            "$ref": "./examples/DeleteWorkspaceManagedSqlServerVulnerabilityAssessment.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/vulnerabilityAssessments": {
+      "get": {
+        "operationId": "WorkspaceManagedSqlServerVulnerabilityAssessments_List",
+        "summary": "Lists the vulnerability assessment policies associated with a server.",
+        "description": "Lists the vulnerability assessment policies associated with a workspace managed sql server.",
+        "tags": [
+          "WorkspaceManagedSqlServer, ServerVulnerabilityAssessments"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./workspace.json#/parameters/workspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved vulnerability assessment policies.",
+            "schema": {
+              "$ref": "#/definitions/ServerVulnerabilityAssessmentListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidStorageAccountName - The provided storage account is not valid or does not exist.\n\n * 400 InvalidStorageAccountCredentials - The provided storage account shared access signature or account storage key is not valid.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 409 UnsupportedWorkspaceType - Unsupported operation for the workspace type.\n\n * 500 InternalServerError - Something went wrong."
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Get workspace managed sql Server's vulnerability assessment policies": {
+            "$ref": "./examples/ListWorkspaceManagedSqlServerVulnerabilityAssessmentPolicies.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/encryptionProtector/{encryptionProtectorName}": {
```

Comment:
please use plurals for resourceType names - even if they are singletons

Reply:
Consistent with SQL Servers Spec and documentation:
![image](https://user-images.githubusercontent.com/72463758/98069269-5a093000-1e84-11eb-85f3-ba41528d89c5.png)

Reply:
resolving it.